src/tag/index.js

```diff
@@ -1,12 +1,19 @@
-import { Component } from 'react';
+import React, { Component } from 'react';
 
 export default class Tag extends Component {
+  render() {
+    const { tag } = this.props;
 
-  getClassNames = () => {
-    return 'pt-tag';
+    return (
+      <li key={tag} className="pt-tag">
+        {tag}
+        <span className="icon" onClick={this._unTag} >&times;</span>
```

Comment:
Yes, but since this was used in referal app, i didn't want to mess with this tag component too much. I can of course do all of these things.